datarobot_provider/example_dags/datarobot_jdbc_dynamic_dataset_dag.py

```diff
@@ -66,4 +66,4 @@ def datarobot_dynamic_jdbc_dataset():
 datarobot_dynamic_jdbc_dataset_dag = datarobot_dynamic_jdbc_dataset()
 
 if __name__ == "__main__":
-    datarobot_dynamic_jdbc_dataset.test()
+    datarobot_dynamic_jdbc_dataset.test()  # type: ignore[attr-defined]
```

Comment:
No way to tell mypy that this callable now have this method

Reply:
My thought was this was a typo and should be:
```suggestion
    datarobot_dynamic_jdbc_dataset_dag.test()  # type: ignore[attr-defined]
```

Reply:
Yeah, this works! Thank you @c-h-russell-walker